sessions/js-basics/survey-error-fixing/js/index.js

```diff
@@ -0,0 +1,30 @@
+console.clear();
+
+// 1. This code selects the "next" button using an ID. This is bad practice. Can you find a better solution?
+const nextButton = document.querySelector('#next');
+nextButton.addEventListener('click', () => {
+	console.log('Go to next question');
+});
+
+// 2. When clicking the "previous" button, this event listener is not executed. Do you know what's wrong with this code?
+const prevButton = document.querySelector('[data-js="prev-button"]');
+prevButton.addEventListener('prev', () => {
+	console.log('Go to previous question');
+});
+
```

Comment:
Maybe add another comment here to distinguish the following two buttons from exercise 2?
```suggestion
// ------------------ 3 & 4 -----------------
```

Reply:
good idea. I also added a comment for 1 & 2 to be consistent